client/src/components/RelatedProducts/ComparisonModal.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper'
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  comparisonHeaderValue: {
+    fontWeight: 700
+  },
+
+  comparisonHeaderBox: {
+    borderBottom: 'solid 2px black',
+    alignItems: 'center'
+  },
+
+  comparisonModal: {
+    position: 'absolute',
+    width: 600,
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    backgroundColor: 'white',
+    padding: '10px'
+  },
+});
+
+const ComparisonModal = React.forwardRef((props, ref) => {
```

Comment:
I'm excited to hear what forwardRef does later!

Reply:
Still figuring it out! But I DO know that there is an error message if you don't use it. : )